Tests/LibWeb/Text/input/wpt-import/media-capabilities/decodingInfo.any.worker.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
```

Comment:
Something seems off here, the expectation file is empty for this test, is something going wrong with the harness?

Reply:
Oh yea, that one times out. Not sure how I didn't notice that.
Removed it now.